### ISSUE TITLE
Bungee Help

### ISSUE THREAD

**Issue by iPintoBean** [kangarko/ChatControl-Pro#971]
I installed the required Bungee plugin and set bungee to true for all servers but when I try to send a PM across servers it says the player is not online.

**Comment by kangarko** [comment]
Please set Debug to [bungee, tell, pm] in settings.yml and post me the output when you send /tell. Also make sure that Private_Messages.Enabled is on true.

**Comment by iPintoBean** [comment]
There was an additional setting I needed to make true. 

I'm having another issue though. I'm using the admin chat across bungee as a staff chat. I gave everyone the proper permissions. It works fine on all servers but one. Every time that server restarts staff chat doesn't work for my staff. Only OP people can talk in it. I have to reload the plugin and then it works again for my staff. Any idea what's causing this?
